client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -50,10 +53,16 @@ $design-selector-selection-space: 285px;
 .design-selector__page-layout-container {
 	position: absolute;
 	top: $design-selector-selection-space;
-	width: 100%;
+	width: calc( 100vw - #{$page-selector-width-offset} );
 	background: $white;
-	box-shadow: $dark-gray-300 0 0 12px;
-	padding: 2em 4em;
+	margin-left: calc( ( ( 100vw - #{$page-selector-width-offset} ) - 100% ) / -2 );
+
+	&.is-open {
+		padding: 2em 4em;
+		box-shadow: $white 0 $page-selector-shadow-size 0 $page-selector-width-offset,
+			$dark-gray-300 0 ( $page-selector-shadow-size - $page-selector-width-offset )
+				$page-selector-shadow-size;
+	}
```

Comment:
Page layout selector container is visible without content when on design selector stage. Setting `visibility: hidden` messes with animation (as the animation had no starting point to animate when visibility is hidden). Quick fix was to do it this way.